script/Ciel/Day 6/06_01_CIEL06_5.txt

```diff
@@ -186,12 +186,12 @@ When I give my compliments to the chef, she replies―――
 -- Page 217, Offset 13715.
 -- 今夜の食材は産地直送、畏れ多くも天皇陛下にもご献上されている逸品ですもの」
 Tonight's ingredients have been sourced fresh from the farm,
- and are considered such delicacies that they've even been served to His Majesty the Emperor."
+ and are considered such delicacies that they've even been served to His Majesty, the Emperor."
 }
 [sha:808d5dd43a2821ed22abcde2f8b04bc68ebbc4ef]{
 -- Page 217, Offset 13716.
 -- なるほど。やはりお高い椎茸さまだったか。
-I see. I didn't realize these were such high-falutin fungi.
+I see. I didn't realize these were such highfalutin fungi.
```

Comment:
I see time crack open before me: a way to the future, death unto our world and all its woes, the shape of things to come. I see... thousands of tweeted screenshots... all of "highfalutin fungi"

> やはりお高い椎茸さまだったか
I didn't realize they were so fancy
I didn't realize they were so upscale.
I didn't realize they were such special shiitake

Reply:
jesus